### PR TITLE
Add Sale price formatting

### PR DESCRIPTION
**Pull request by milofultz**

### CODE REVIEW

File: client/src/components/ProductDetailPage.jsx [hunk 1]
```diff
@@ -26,8 +26,6 @@ class ProductDetailPage extends React.Component {
       currentProductId: 22126, // this should be set at runtime by the productId in the url? or if none given, has a default
       relatedProducts: [],
       products: {
-        // mock products until successful API calls
-        '22126': productsMock
```
Comment (NylonEric) [inline client/src/components/ProductDetailPage.jsx:30]: I see you are removing the default product. Is there handling for the exception in the components if the API fails?

**Review by NylonEric — APPROVED** [review]
Approved, but keep in mind the non-ideal case of one or more APIs failing.

File: client/src/components/ProductDetailPage.jsx (hunk 1 above)
Comment (milofultz) [reply client/src/components/ProductDetailPage.jsx:30]: The problem is that the default product does not match the product we are pulling in from our API and makes the app crash.

Comment (milofultz) [reply client/src/components/ProductDetailPage.jsx:30]: I guess we could fix the mock

**Pull request merged**